tests/test_benchmark.py

```diff
@@ -156,7 +156,7 @@ def test_benchmark_checksum(is_single_task, test_single_task_benchmark, test_mul
 
     # --- Test that the checksum is NOT the same ---
     def _check_for_failure(_kwargs):
-        with pytest.raises(ValidationError) as error:
+        with pytest.raises((ValidationError, TypeError)) as error:
```

Comment:
Why is this needed? Seems unrelated!

Reply:
In the failure tests, a column of strings was tested. That string array fails at the `np.isnan` operator due to the type. 
Both TypeError and ValidationError are raised in that case.So the TypeError has to be added here.